python/pyspark/sql/tests/test_functions.py

```diff
@@ -962,6 +962,11 @@ def test_lit_day_time_interval(self):
         actual = self.spark.range(1).select(lit(td)).first()[0]
         self.assertEqual(actual, td)
 
+    def test_lit_list(self):
+        test_list = [1, 2, 3]
```

Comment:
Should probably have one more test with other fixed types. e.g.) `["a", 1, None, 1.0]`